flytekit/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 import flytekit.plugins  # noqa: F401
 
-__version__ = "0.13.0b13"
+__version__ = "0.12.11"
```

Comment:
let's keep the minor and make a beta?  if you want to use 0.12.x can you make the same changes to the `release-0.12` branch?  Thanks.